src/SV0001Itinerary/Utils/api/DB0002Itineraries.ts

```diff
@@ -0,0 +1,45 @@
+import db from '../fireB/firestore'
+import { doc, collection, getDoc, setDoc, addDoc, deleteDoc, getDocs } from 'firebase/firestore';
+import type { DB0002ItineraryType } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
+import { initItinerary } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
+
+export const collectionName = 'Itineraries';
+const collectionRef = collection(db, collectionName);
+
+export const read = async (id: string): Promise<DB0002ItineraryType | null> => {
```

Comment:
前回の打ち合わせでも話したけど、個人的には↓みたいにしたいんだよね、、、
string の方が可読性が良いなら、itineraryID: string でも良いかもね（DBxxxxhogeで統一できなくなるけど、、、）
```suggestion
export const read = async (id: DB0002ItineraryType['id']): Promise<DB0002ItineraryType | null> => {
```

Reply:
これどうすればいいのかわからなくて一旦このままにしてた